components/brave_wallet_ui/common/async/lib.ts

```diff
@@ -862,26 +838,15 @@ export function refreshKeyringInfo () {
     const apiProxy = getAPIProxy()
     const { keyringService, walletHandler, jsonRpcService } = apiProxy
     const walletInfoBase = (await walletHandler.getWalletInfo()).walletInfo
-    const walletInfo = { ...walletInfoBase, visibleTokens: [], selectedAccount: '' }
+    const { allAccounts } = (await apiProxy.keyringService.getAllAccounts())
+    const walletInfo = { ...walletInfoBase, accountInfos: allAccounts.accounts, selectedAccount: '' }
 
     // Get/Set selectedAccount
     if (!walletInfo.isWalletCreated) {
       dispatch(WalletActions.initialized(walletInfo))
       return
     }
 
-    // Get default accounts for each CoinType
-    const defaultAccounts = await Promise.all(SupportedCoinTypes.map(async (coin: BraveWallet.CoinType) => {
-      const { chainId } = await jsonRpcService.getDefaultChainId(coin)
-      const defaultAccount = coin === BraveWallet.CoinType.FIL
-        ? await keyringService.getFilecoinSelectedAccount(chainId)
-        : await keyringService.getSelectedAccount(coin)
-      const defaultAccountAddress = defaultAccount.address
-      return walletInfo.accountInfos.find((account) => account.address.toLowerCase() === defaultAccountAddress?.toLowerCase()) ?? {} as BraveWallet.AccountInfo
-    }))
-    const filteredDefaultAccounts = defaultAccounts.filter((account) => Object.keys(account).length !== 0)
-    dispatch(WalletActions.setDefaultAccounts(filteredDefaultAccounts))
-
```

Comment:
defaultAccounts stuff was not used and confirmed with @josheleonard to be removed